internal/ingress/metric/collectors/controller.go

```diff
@@ -204,6 +214,26 @@ func (cm *Controller) IncCheckErrorCount(namespace, name string) {
 	cm.checkIngressOperationErrors.MustCurryWith(cm.constLabels).With(labels).Inc()
 }
 
+// IncOrphanIngress sets the the orphaned ingress gauge to one
+func (cm *Controller) IncOrphanIngress(namespace string, name string, type_ string) {
```

Comment:
why this as type_? can't we use just "type" here?

Reply:
I thought to write it like this because `type` is golang keyword, to clarify that it is a variable name, not  a keyword

Reply:
I can change variable name to smth like `orphanityType`

Reply:
i like the more specific name orphanityType so there is no confusing in the future.